mbuild/tests/test_par.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pytest
+
+import mbuild as mb
+from mbuild.tests.base_test import BaseTest
+from mbuild.utils.io import get_fn
+from mbuild.utils.io import has_foyer
+from mbuild.formats.par_writer import write_par
+
+
+@pytest.mark.skipif(not has_foyer, reason="Foyer package not installed")
+class TestPar(BaseTest):
+
+    @pytest.mark.skipif(not has_foyer, reason="Foyer package not installed")
+    def test_save_charmm(self):
+        cmpd = mb.load(get_fn('charmm_dihedral.mol2'))
+        for i in cmpd.particles():
+            i.name = "_{}".format(i.name)
+        structure = cmpd.to_parmed(box=cmpd.boundingbox, 
+                                    residues=set([p.parent.name for \
+                                                 p in cmpd.particles()]))
+
+        from foyer import Forcefield
+        ff = Forcefield(forcefield_files=[get_fn('charmm_truncated.xml')])
+        structure = ff.apply(structure, assert_dihedral_params=False)
+
+        write_par(structure, 'charmm_dihedral.par')
```

Comment:
I'm confident that the saved .par file is correct, but this test seem to actually test that. You should add an assertion to validate the output file somehow (i.e. number of lines or if we wanted to be more thorough, we could reload the saved .par file using Parmed and compare bonds/atoms to `ff`)

Reply:
Okay there's a new test that uses parmed to load in the parameters and just checks for the numbers of different parameters present, but doesn't do any numerical comparison or validation

Reply:
That works for me